runtime/onert/core/src/loader/traininfo_loader.cc

```diff
@@ -62,6 +62,7 @@ ir::train::LossInfo loadLossInfo(const circle::ModelTraining *circle_model)
   // fill ir_loss from circle_loss
   ir::train::LossInfo ir_loss;
   const circle::LossFn circle_loss = circle_model->lossfn();
+  const circle::LossReductionType circle_loss_rdt = circle_model->loss_reduction_type();
```

Comment:
Q: why this change happened? 

From this PR - https://github.com/Samsung/ONE/pull/12607, train info schema updated to support loss reduction type.
So, Let's update the loader to support loss reduction type.